examples/complete-hosted-zone-in-diff-aws/README.md

```diff
@@ -0,0 +1,47 @@
+# Complete ACM example with Route53 DNS validation in another AWS account
+
+Configuration in this directory creates new Route53 zone in another AWS account with provided credentials and ACM certificate (valid for the domain name and wildcard) in current account.
```

Comment:
It looks like this example is not creating a new Route 53 zone. [Line 14](https://github.com/terraform-aws-modules/terraform-aws-acm/blob/5824abf672b5ced4be2521502f5c82b00e3ffabf/examples/complete-hosted-zone-in-diff-aws/main.tf#L14) of the example's `main.tf` has `use_existing_route53_zone = true`.

Reply:
Yes, Route 53 zone can pre-exists and thus for DNS validation `CNAME` entries need to be created using `acm` module.

The client can create the hosted zone by using `use_existing_route53_zone = false` and that will be depend on use case of the client!